### ISSUE TITLE
"data_interpretations" in step-68 is un-used

### ISSUE THREAD

**Issue by ZhouleiJoeStone** [dealii/dealii#11252]
un-used code in step-68:

```
template <int dim>
void ParticleTracking<dim>::output_particles(const unsigned int it)
{
  Particles::DataOut<dim, dim> particle_output;
  std::vector<std::string> solution_names(dim, "velocity");
  solution_names.push_back("process_id");
  std::vector<DataComponentInterpretation::DataComponentInterpretation>
    data_interpretations;  // **<-- this line is un-used**
  std::vector<DataComponentInterpretation::DataComponentInterpretation>
    data_component_interpretation(
      dim, DataComponentInterpretation::component_is_part_of_vector);
  data_component_interpretation.push_back(
    DataComponentInterpretation::component_is_scalar);
....

} 
```

**Comment by bangerth** [comment]
@ZhouleiJoeStone -- yes, excellent observation! I suspect this is just a remnant of the development process.

Would you like to give it a try to fix this? There is a good list of instructions here that illustrate the process: https://github.com/dealii/dealii/wiki/Contributing This would then also make you one of the authors of deal.II!

**Comment by ZhouleiJoeStone** [comment]
@bangerth 

In [github's step-68.cc](https://github.com/dealii/dealii/blob/master/examples/step-68/step-68.cc), I use the edit pencil buttom on webpage to delete these un-used two lines

``` 
std::vector<DataComponentInterpretation::DataComponentInterpretation>	
     data_interpretations; 
```

I think this pull request will work.

**Comment by kronbichler** [comment]
Fixed by #11265. Thank you!
